go.mod

```diff
@@ -1,3 +1,16 @@
-module github.com/turnage/graw
+module github.com/monkeydioude/graw
 
 go 1.13
+
+require (
```

Comment:
If these changes are required, they should take place in a separate PR.

Reply:
Ok